lib/easy-stackup.js

```diff
@@ -0,0 +1,72 @@
+//simpler API main function
+'use strict'
+var gerberToSvg = require('gerber-to-svg')
+var shortId = require('shortid')
+var whatsThatGerber = require('whats-that-gerber')
+
+var pcbStackup = require('./index')
+
+module.exports = function(layers, optionsOrCallback, callback) {
```

Comment:
I generally prefer the callback function to be called "done" because I find it more descriptive